Assets/MixedRealityToolkit/Attributes/MixedRealityExtensionServiceAttribute.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions;
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Utilities;
+using Microsoft.MixedReality.Toolkit.Core.Utilities.Editor.Setup;
+using System;
+using System.Linq;
+using UnityEngine;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Core.Attributes
+{
+    /// <summary>
+    /// Attribute that defines the properties of a Mixed Reality Toolkit extension service.
+    /// </summary>
+    [AttributeUsage(System.AttributeTargets.Class, AllowMultiple = false)]
+    public class MixedRealityExtensionServiceAttributeAttribute : Attribute
```

Comment:
`MixedRealityExtensionServiceAttributeAttribute`? Two `Attributes` seems strange

Reply:
that looks like a typo.... let me check